kong/hooks.lua

```diff
@@ -8,6 +8,7 @@ local ipairs = ipairs
 local pack = table.pack
 local unpack = table.unpack
 local insert = table.insert
+local EMPTY = require("pl.tablex").readonly({})
```

Comment:
According the discussion in #11443, `EMPTY` is not necessary.

Reply:
In the internal profiling, it create too many tmp table during the proxy.

Reply:
Could you leave a comment in #11443? Perhaps we can reopen that PR.